base/changes.txt

```diff
@@ -6,6 +6,10 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2022-07-10  David Carlisle  <David.Carlisle@latex-project.org>
+
+	* cmfonts.fdd: avoid EC ss fonts at small design sizes. Scale 8pt font. gh/879
```

Comment:
should say why eg  , because they are broken            or something

Reply:
I'll add some words